micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/ServiceRestClientConfigurationSupport.groovy

```diff
@@ -120,8 +122,17 @@ class ServiceRestClientConfigurationSupport {
     }
 
     @Bean
-    MetricsAspect metricsAspect(MetricRegistry metricRegistry) {
-        return new MetricsAspect(metricRegistry)
+    @ConditionalOnExpression('${rest.client.metrics.enabled:true}')
+    RestOperationsMetricsAspect restOperationsMetricsAspect(
+            MetricRegistry metricRegistry, URIMetricNamer uriMetricNamer
+    ) {
+        return new RestOperationsMetricsAspect(metricRegistry, uriMetricNamer)
+    }
+
+    @Bean
+    @ConditionalOnMissingBean(URIMetricNamer)
```

Comment:
Just tested it with boot-microservice and it works as expected. Why would we test that manually? It's Spring's feature after all...